object_database/web/content/CellHandler.js

```diff
@@ -110,6 +135,61 @@ class CellHandler {
         }
     }
 
+    /**
+     * Message that tells the handler to append
+     * a specific and immediate script to
+     * the postscripts list.
+     */
+    appendPostscript(message){
+        this.postscripts.push(message.script);
+    }
+
+    /**
+     * Message that tells us a Cell has been
+     * discarded and is no longer needed.
+     * We remove both it's DOM representation
+     * and any registered components for it.
+     */
+    cellDiscarded(message){
+        let foundCell = this.cells[message.id];
+        let foundComponent = this.activeComponents[message.id];
+
+        if(foundCell == undefined){
+            console.warn(`Received discard message for non-existing Cell ${message.id}`);
+        }
+
+        if(foundComponent == undefined){
+            console.warn(`Received discard message for non-existing Component ${message.id}`);
+        }
+
+        // If the cell has a domNode property,
+        // it means it's a hyperscript element
+        // and this is what we should deal with.
+        if(foundCell && foundCell.domNode !== undefined){
+            foundCell = foundCell.domNode;
+        }
+
+        if(foundComponent){
```

Comment:
do we need to remove `this.cells[message.id]` as well?

Reply:
We need to keep it around for now. Here's why:
* `this.activeComponents` is a dictionary that stores component instances by CellID
* `this.cells` is a dictionary that stores *rendered* hyperscript or DOM elements by CellID.
  
We currently still need both because the projector needs to know where to replace things.